app/routes/_layout+/calendar.tsx

```diff
@@ -110,6 +110,55 @@ const Calendar = () => {
     [ripple]
   );
 
+  type BookingStatus =
+    | "DRAFT"
+    | "ARCHIVED"
+    | "CANCELLED"
+    | "RESERVED"
+    | "ONGOING"
+    | "OVERDUE"
+    | "COMPLETE";
+
+  const statusClassesOnHover: Record<BookingStatus, string> = {
```

Comment:
You should make this work with tailwind classNames rather than with hardcoded colors. On the hover events you can add and remove classes. If we do it this way once we change the colors of the tailwind theme we would have to update it here and I dont wnat to do that. it should be managed in 1 central place. In that case you also dont need 2 objects, you can have 1 object with classes.